tobler/dasymetric/masked_area_interpolate.py

```diff
@@ -1,18 +1,20 @@
-from ..area_weighted import _slow_area_interpolate, _area_tables_raster
+import geopandas as gpd
 
-from ..area_weighted._vectorized_raster_interpolation import *
+from ..area_weighted import area_interpolate
+from .raster_tools import extract_raster_features
 
 
 def masked_area_interpolate(
     source_df,
     target_df,
-    raster="nlcd_2011",
-    codes=None,
-    force_crs_match=True,
+    raster,
+    pixel_values=[21, 22, 23, 24],
```

Comment:
Shall we actually have a default here? This is specific NLCD data. While I understand that it was the dataset you often use with this function, it is by no means the expected dataset.

Reply:
Also, this is a breaking change if we change `codes` to `pixel_values` that should have some deprecation period.

Reply:
+2